candi/cloud-providers/yandex/openapi/cluster_configuration.yaml

```diff
@@ -382,10 +383,12 @@ apiVersions:
             description: |
               The cloud ID.
             type: string
+            x-unsafe: true
```

Comment:
x-unsafe set only for yandex. Is it correct?

Reply:
Added unsafe-validations for other providers as well.